### PR TITLE
Color schemes pass 2

### LINKED ISSUE #193: Color scheme shadow info and typo
Hi,

I realize this documentation isn't maintained at the moment, but thought I would log this for people that are interested.

The text at http://docs.sublimetext.info/en/latest/reference/color_schemes.html#shadow isn't very descriptive.

It seems that the shadow is visible when the text in the document is wider than the window, and will appear on the left and/or the right, depending whether the view can be scrolled to the left or the right.

Also, there is a typo at `shadowWidth`

> Width o**t** the shadow effect when the buffer is scrolled.

and setting it like this:

```
<key>shadowWidth</key>
<string>42</string>
```

seems to remove the shadow altogether, so maybe some example values could be provided? Otherwise it seems that the shadow only appears when this key doesn't exist in the tmTheme file.

### PR DESCRIPTION
**Pull request by FichteFoll**
Couple fixes and revisions of the color scheme docs.

I was working on a color scheme and found the misleading docs very annoying, so I just fixed a couple issues and worked myself through remaining comments on #130. Appropriate tasks are checked in the trello card.

Fixes #193.

### CODE REVIEW

File: source/reference/color_schemes.rst [hunk 1]
```diff
@@ -65,9 +67,11 @@ by means of the **Preferences → Color Scheme** menu.
 A common way of selecting a color scheme
 is by associating it to a type of file
 using the file-type-specific settings.
-For example, for the Python file type (``syntax_file`` == :file:`Python.tmLanguage`),
-we'd use the :file:`Python.sublime-settings` file
-and, within, set ``color_scheme`` to some color scheme file.
+Select **Preferences → Settings - Syntax Specific**
+to open the settings file
```
Comment (guillermooo) [inline source/reference/color_schemes.rst]: above we say `file-type-specific`settings and now it seems we talk about something different. Consider using `file-type-specific` again.

File: source/reference/color_schemes.rst [hunk 2]
```diff
@@ -65,9 +67,11 @@ by means of the **Preferences → Color Scheme** menu.
...
+of the currently active syntax
+and set the ``color_scheme`` setting
+to some color scheme file.
```
Comment (guillermooo) [inline source/reference/color_schemes.rst]: => *to the name of a color scheme file*

File: source/reference/color_schemes.rst [hunk 3]
```diff
@@ -115,14 +119,19 @@ Topmost Elements in Color Schemes Files
    Name of the color scheme.
    Ignored by Sublime Text.
 
+``settings``
+   Required.
+   Container for further color scheme settings.
+   See the section below for details.
```
Comment (guillermooo) [inline source/reference/color_schemes.rst]: => *next* section ??

File: source/reference/color_schemes.rst [hunk 4]
```diff
@@ -155,18 +164,23 @@ within the topmost ``<array>``.
...
-Global Settings Ordered by Type
--------------------------------
-
-
 General
 ^^^^^^^
 
 ``foreground``
-   Foreground color for the view.
+   Standard foreground color for the view.
```
Comment (guillermooo) [inline source/reference/color_schemes.rst]: Standard => *Default* ??

File: source/reference/color_schemes.rst [hunk 5]
```diff
@@ -155,18 +164,23 @@ within the topmost ``<array>``.
...
+
+   Because there is no override setting for rulers,
+   the only way to change the color of rulers
+   is a "hack" further described `here`__.
+
+   .. __: https://github.com/icylace/CursorRuler/wiki/Tips#ruler-colors
 
 ``background``
-   Backgound color of the view.
+   Background color of the view (and gutter).
```
Comment (guillermooo) [inline source/reference/color_schemes.rst]: => *Default* background...

File: source/reference/color_schemes.rst (hunk 2 above)
Comment (FichteFoll) [reply source/reference/color_schemes.rst]: It's not about names but about paths, though.

File: source/_includes/color_scheme_summary_table.g.txt [hunk 6]
```diff
@@ -0,0 +1,13 @@
+.. This file was generated. Don't edit it.
```
Comment (guillermooo) [inline source/_includes/color_scheme_summary_table.g.txt:1]: Cool! :)

**Review by guillermooo — APPROVED** [review]

**Review by guillermooo — APPROVED** [review]
GH review system is rather confusing... I don't think I'm using it right, but LGTM!

**Pull request merged**